packages/test/test-service-load/src/testConfigFile.ts

```diff
@@ -69,6 +69,16 @@ export interface ILoadTestConfig {
      * Specify Ops payload size for the test run.
      */
     opSizeinBytes?: number;
+    /**
+     * Whether the ops generated should have random content.
+     * This config can be used to force a low compression ratio.
+     */
+    useRandomContent?: boolean;
+    /**
+     * If enabled, for each op, the test will generate content of random size
+     * between 0 and `opSizeinBytes`
+     */
+    useVariableOpSize?: boolean;
```

Comment:
Can we not add these options to config, and always just use _true_ in code? Stress tests are not deterministic. If you want clients to behave differently, I'd rather put it into runtimeOptionsMatrix, but enumerate all these differences within a single test run, vs. requiring running many test runs to hit different configs. 
Maybe other way to say it - most options here ideally are used for differences in external test structure, i.e. single machine (our tests) vs. 100 machines and thousands of clients in ODSP stress tests. Stressing system till it fails vs. stress workload that does not reach such level, but tests system under pressure.

Reply:
Not sure I follow, @vladsud. Are you saying stress test clients should always send ops with random sizes? Clients do behave differently based on the runtimeOpionsMatrix, but that's governing internal runtime behavior, not client behavior (like frequency of ops/signals, size of ops, blob counts, etc..).

Reply:
Yes, variable sizes all the time. Basically, it's not scalable solution to keep adding to this config - at some point we will need to run 1000 different test runs to validate full permutation of this config, OR (more realistically) - never use some of the options. I think better approach is to have one run (ore very few runs) and bake all interesting permutations we need to test into that single run via runtimeOptionsMatrix, leaving this config mostly reflect differences between different sub-systems (our stress tests vs. ODSP stress tests vs. maybe FRS specific runs).

Reply:
Got it. Yes, that's my view as well. I want to first minimize the risk here as the stress test pipeline has already been strained by other changes and this may create more churn. The goal is to get rid of this and unify the pipelines/tests.

Reply:
As long as there is a follow-up task, I'm fine with that :)